### DISCUSSION TITLE
Group Table Bulk Actions

### DISCUSSION THREAD

**Discussion by LooxisDev** [filamentphp/filament#4940] [Ideas]
In some cases I would love to have a subnavigation for the Table bulk actions to structure or group them.

Currently I have a table with > 10 Actions and there is no default way of grouping them like for the single ActionGroups.

It would be awesome to group BulkActions in groups, too. So I could for example give a group a specific permission etc.

Here is a small example for an invoices table:

Actions:

- Generate PDF
- Send Mail
- Download Invoices (Zip)
- Recalculate amount/taxes
-Upload to Amazon
- Transfer to AccountingSoftware
...

It would be awesome if it could be grouped like:

- General Actions
---- Genearte PDF
---- Download

- Mail
---- Send Mail

- Accounting
---- Recalculate
---- Transfer to AccountingSoftware

- Platforms
---- Upload to Amazon

**Comment by danharrin** [comment]
Accepted feature request - #4950
